src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -764,14 +763,10 @@ static unique_ptr<Scene> _useBuildHelper(SvgLoaderData& loaderData, const SvgNod
             }
             viewBoxClip->transform(mClipTransform);
 
-            auto compositeLayer = Scene::gen();
-            compositeLayer->composite(std::move(viewBoxClip), CompositeMethod::ClipPath);
-            compositeLayer->push(std::move(scene));
-
-            auto root = Scene::gen();
-            root->push(std::move(compositeLayer));
+            scene->clip(std::move(viewBoxClip));
 
-            finalScene = std::move(root);
+            finalScene = Scene::gen();
+            finalScene->push(std::move(scene));
```

Comment:
I don't think `finalScene` is necessary since the composition work for the scene has been simplified.
How about remove `finalScene`? When I tested it, the result was visually fine.

```diff
--- a/src/loaders/svg/tvgSvgSceneBuilder.cpp
+++ b/src/loaders/svg/tvgSvgSceneBuilder.cpp
@@ -713,7 +713,6 @@ static Matrix _calculateAspectRatioMatrix(AspectRatioAlign align, AspectRatioMee

 static unique_ptr<Scene> _useBuildHelper(SvgLoaderData& loaderData, const SvgNode* node, const Box& vBox, const string& svgPath, int depth, bool* isMaskWhite)
 {
-    unique_ptr<Scene> finalScene;
     auto scene = _sceneBuildHelper(loaderData, node, vBox, svgPath, false, depth + 1, isMaskWhite);

     // mUseTransform = mUseTransform * mTranslate
@@ -750,9 +749,7 @@ static unique_ptr<Scene> _useBuildHelper(SvgLoaderData& loaderData, const SvgNod
         mSceneTransform = mUseTransform * mSceneTransform;
         scene->transform(mSceneTransform);

-        if (node->node.use.symbol->node.symbol.overflowVisible) {
-            finalScene = std::move(scene);
-        } else {
+        if (!node->node.use.symbol->node.symbol.overflowVisible) {
             auto viewBoxClip = Shape::gen();
             viewBoxClip->appendRect(0, 0, width, height, 0, 0);

@@ -764,16 +761,12 @@ static unique_ptr<Scene> _useBuildHelper(SvgLoaderData& loaderData, const SvgNod
             viewBoxClip->transform(mClipTransform);

             scene->clip(std::move(viewBoxClip));
-
-            finalScene = Scene::gen();
-            finalScene->push(std::move(scene));
         }
     } else {
         scene->transform(mUseTransform);
-        finalScene = std::move(scene);
     }

-    return finalScene;
+    return scene;
 }

```